tests/core/consensus/test_block_creation.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+import random
+from functools import reduce
+from typing import List
+
+import pytest
+
+from chia.consensus.block_creation import compute_block_cost, compute_block_fee
+from chia.consensus.default_constants import DEFAULT_CONSTANTS
+from chia.types.blockchain_format.coin import Coin
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.util.ints import uint32
+from tests.core.make_block_generator import make_block_generator
+
+
+def rand_hash() -> bytes32:
+    rng = random.Random()
+    ret = bytearray(32)
+    for i in range(32):
+        ret[i] = rng.getrandbits(8)
+    return bytes32(ret)
+
+
+@pytest.mark.parametrize("add_amount", [[0], [1, 2, 3], []])
+@pytest.mark.parametrize("rem_amount", [[0], [1, 2, 3], []])
+def test_compute_block_fee(add_amount: List[int], rem_amount: List[int]) -> None:
+    additions: List[Coin] = [Coin(rand_hash(), rand_hash(), amt) for amt in add_amount]
+    removals: List[Coin] = [Coin(rand_hash(), rand_hash(), amt) for amt in rem_amount]
+
+    # the fee is the left-overs from the removals (spent) coins after deducting
+    # the newly created coins (additions)
+    expected = reduce(lambda a, b: a + b, rem_amount, 0) - reduce(lambda a, b: a + b, add_amount, 0)
+
+    if expected < 0:
+        with pytest.raises(ValueError, match="does not fit into uint64"):
+            compute_block_fee(additions, removals)
+    else:
+        assert compute_block_fee(additions, removals) == expected
+
+
+def test_compute_block_cost(softfork_height: uint32) -> None:
+    num_coins = 10
+    generator = make_block_generator(num_coins)
+    cost = int(compute_block_cost(generator, DEFAULT_CONSTANTS, softfork_height))
+
+    coin_cost = 1800000 * num_coins
+    agg_sig_cost = 1200000 * num_coins
+
+    cost -= coin_cost
+    cost -= agg_sig_cost
+    print(cost)
```

Comment:
Maybe more than this would be good, but at least have some sort of text with the output.

```suggestion
    print(f"{cost=}")
```